Header header

#A position 6D (x,y,z,wx,wy,wz)
float32[] position_right_arm
float32[] position_left_arm

